RSConnectDIOToSnap7/RSConnectDIOToSnap7.en.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LibraryResource xmlns="urn:abb-robotics-robotstudio-libraryresource"
                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                 xsi:schemaLocation="urn:abb-robotics-robotstudio-libraryresource file:///C:\Program%20Files%20(x86)\ABB%20Industrial%20IT\Robotics%20IT\SDK\RobotStudio%20SDK%206.07\LibraryResourceSchema.xsd">
+                 xsi:schemaLocation="urn:abb-robotics-robotstudio-libraryresource file:///C:\Program%20Files%20(x86)\ABB%20Industrial%20IT\Robotics%20IT\SDK\RobotStudio%20SDK%206.08\LibraryResourceSchema.xsd">
 	<SmartComponent name="RSConnectDIOToSnap7" description="Allow to connect Robot with real or simulated S7 PLC using Sharp7(Snap7).">
 		<DynamicProperty name="Status" description="The status of this component."/>
 		<DynamicProperty name="PLC_Addr" description="IP Address of PLC IPV4.

```

Comment:
Could you change description of Dx_Number to "Number of PLC Dx to connect to in Station Logic..."